pkg/routes/api/v1/user_export.go

```diff
@@ -126,14 +129,25 @@ func DownloadUserDataExport(c echo.Context) error {
 		return handler.HandleHTTPError(err)
 	}
 
+	// Check if user has an export file
+	if u.ExportFileID == 0 {
+		return echo.NewHTTPError(http.StatusNotFound, "No user data export found.")
+	}
+
 	// Download
 	exportFile := &files.File{ID: u.ExportFileID}
 	err = exportFile.LoadFileMetaByID()
 	if err != nil {
+		if files.IsErrFileDoesNotExist(err) {
+			return echo.NewHTTPError(http.StatusNotFound, "User data export file not found.")
```

Comment:
Please use the same error as in line 134. You can put it in a variable and return the same error in all cases here